purview/app.py

```diff
@@ -0,0 +1,13 @@
+# SPDX-License-Identifier: GPL-3.0+
+
+from flask import Flask
+
+from purview.logger import init_logging
+
+
+def create_app():
+    app = Flask(__name__)
+    # Initialize logging
```

Comment:
Bike-shed: This comment isn't useful since it's clear this is happening in the line below.